FrontEnd/src/components/adminPage/detail-view/DeleteModal.jsx

```diff
@@ -0,0 +1,29 @@
+import css from "./DeleteModal.module.css"
+
+function DeleteModal({active, setActive}) {
+
+    const onDeleteClick = () => {
+        console.log("Delete")
+    }
+
+    return !active ? null : (
+        <div
```

Comment:
I would recommend creating a separate component `Modal` that could be reusable. Also, we can utilize components from [ant design](https://ant.design/components/modal), since designers used their style kit. But as far as see we don't look for easy ways